common/v2/features/AddAccount/__tests__/AddAccountFlow.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { MemoryRouter, Route, Switch } from 'react-router';
+import { simpleRender, fireEvent } from 'test-utils';
+
+import { AccountContext, NetworkContext, INetworkContext } from 'v2/services/Store';
+import { translateRaw } from 'v2/translations';
+import { ROUTE_PATHS, WALLETS_CONFIG } from 'v2/config';
+import { WalletId } from 'v2/types';
+import AddAccountFlow, { isValidWalletId } from 'v2/features/AddAccount/AddAccountFlow';
+import { IAccountContext } from 'v2/services/Store/Account/AccountProvider';
+
+/* Test helpers */
+describe('isValidWalletId()', () => {
+  it('Determines if a string is a valid WalletId', () => {
+    expect(isValidWalletId('WEB3')).toBe(true);
+    expect(isValidWalletId('web3')).toBe(false);
```

Comment:
Based on the name, i'd have considered 'web3' to be a `true` `isValidWalletId()`. Since it's not, i'm assuming this check is done after the key is cast as `toUpperCase()`?

Reply:
Yes, it's to make sure we call toUpperCase before we validate.